{
  "uid" : "16015c77b960bd4e",
  "name" : "Verify that the 'Logo' link on the Start Authorized Page \n                             doesn't refresh the current page or lead to other pages after clicking",
  "fullName" : "tests.header_test.TestAuthHeaderPageLinks#test_hpa_03_04_verify_click_auth_logo_link",
  "historyId" : "9853c8de8d2e73fdb84d42921d4b6b68",
  "time" : {
    "start" : 1728694593458,
    "stop" : 1728694593458,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56461dd3b02a <unknown>\n#1 0x56461da215e0 <unknown>\n#2 0x56461da197eb <unknown>\n#3 0x56461da09a09 <unknown>\n#4 0x56461da0b744 <unknown>\n#5 0x56461da09cbd <unknown>\n#6 0x56461da0952f <unknown>\n#7 0x56461da09422 <unknown>\n#8 0x56461da07493 <unknown>\n#9 0x56461da07ada <unknown>\n#10 0x56461da23f41 <unknown>\n#11 0x56461dab58c5 <unknown>\n#12 0x56461da95b22 <unknown>\n#13 0x56461dab4d7d <unknown>\n#14 0x56461da958c3 <unknown>\n#15 0x56461da636b3 <unknown>\n#16 0x56461da6468e <unknown>\n#17 0x56461dd05a2b <unknown>\n#18 0x56461dd099b1 <unknown>\n#19 0x56461dcf2225 <unknown>\n#20 0x56461dd0a532 <unknown>\n#21 0x56461dcd738f <unknown>\n#22 0x56461dd29f28 <unknown>\n#23 0x56461dd2a0f3 <unknown>\n#24 0x56461dd39e7c <unknown>\n#25 0x7fa5c4694ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"8cbc0798f0020718cedcbad3368d96dd\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f8f5b304c20>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x56461dd2a0f3 \\\\u003Cunknown>\\\\n#24 0x56461dd39e7c \\\\u003Cunknown>\\\\n#25 0x7fa5c4694ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x56461dd3b02a <unknown>\nE       #1 0x56461da215e0 <unknown>\nE       #2 0x56461da197eb <unknown>\nE       #3 0x56461da09a09 <unknown>\nE       #4 0x56461da0b744 <unknown>\nE       #5 0x56461da09cbd <unknown>\nE       #6 0x56461da0952f <unknown>\nE       #7 0x56461da09422 <unknown>\nE       #8 0x56461da07493 <unknown>\nE       #9 0x56461da07ada <unknown>\nE       #10 0x56461da23f41 <unknown>\nE       #11 0x56461dab58c5 <unknown>\nE       #12 0x56461da95b22 <unknown>\nE       #13 0x56461dab4d7d <unknown>\nE       #14 0x56461da958c3 <unknown>\nE       #15 0x56461da636b3 <unknown>\nE       #16 0x56461da6468e <unknown>\nE       #17 0x56461dd05a2b <unknown>\nE       #18 0x56461dd099b1 <unknown>\nE       #19 0x56461dcf2225 <unknown>\nE       #20 0x56461dd0a532 <unknown>\nE       #21 0x56461dcd738f <unknown>\nE       #22 0x56461dd29f28 <unknown>\nE       #23 0x56461dd2a0f3 <unknown>\nE       #24 0x56461dd39e7c <unknown>\nE       #25 0x7fa5c4694ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728694482412,
      "stop" : 1728694482412,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1728694593459,
      "stop" : 1728694594237,
      "duration" : 778
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728694594237,
      "stop" : 1728694594870,
      "duration" : 633
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56461dd3b02a <unknown>\n#1 0x56461da215e0 <unknown>\n#2 0x56461da197eb <unknown>\n#3 0x56461da09a09 <unknown>\n#4 0x56461da0b744 <unknown>\n#5 0x56461da09cbd <unknown>\n#6 0x56461da0952f <unknown>\n#7 0x56461da09422 <unknown>\n#8 0x56461da07493 <unknown>\n#9 0x56461da07ada <unknown>\n#10 0x56461da23f41 <unknown>\n#11 0x56461dab58c5 <unknown>\n#12 0x56461da95b22 <unknown>\n#13 0x56461dab4d7d <unknown>\n#14 0x56461da958c3 <unknown>\n#15 0x56461da636b3 <unknown>\n#16 0x56461da6468e <unknown>\n#17 0x56461dd05a2b <unknown>\n#18 0x56461dd099b1 <unknown>\n#19 0x56461dcf2225 <unknown>\n#20 0x56461dd0a532 <unknown>\n#21 0x56461dcd738f <unknown>\n#22 0x56461dd29f28 <unknown>\n#23 0x56461dd2a0f3 <unknown>\n#24 0x56461dd39e7c <unknown>\n#25 0x7fa5c4694ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728694594237,
        "stop" : 1728694594869,
        "duration" : 632
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56461dd3b02a <unknown>\n#1 0x56461da215e0 <unknown>\n#2 0x56461da197eb <unknown>\n#3 0x56461da09a09 <unknown>\n#4 0x56461da0b744 <unknown>\n#5 0x56461da09cbd <unknown>\n#6 0x56461da0952f <unknown>\n#7 0x56461da09422 <unknown>\n#8 0x56461da07493 <unknown>\n#9 0x56461da07ada <unknown>\n#10 0x56461da23f41 <unknown>\n#11 0x56461dab58c5 <unknown>\n#12 0x56461da95b22 <unknown>\n#13 0x56461dab4d7d <unknown>\n#14 0x56461da958c3 <unknown>\n#15 0x56461da636b3 <unknown>\n#16 0x56461da6468e <unknown>\n#17 0x56461dd05a2b <unknown>\n#18 0x56461dd099b1 <unknown>\n#19 0x56461dcf2225 <unknown>\n#20 0x56461dd0a532 <unknown>\n#21 0x56461dcd738f <unknown>\n#22 0x56461dd29f28 <unknown>\n#23 0x56461dd2a0f3 <unknown>\n#24 0x56461dd39e7c <unknown>\n#25 0x7fa5c4694ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"8cbc0798f0020718cedcbad3368d96dd\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728694594901,
      "stop" : 1728694594963,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Header"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "header_test"
  }, {
    "name" : "subSuite",
    "value" : "TestHeaderPage"
  }, {
    "name" : "host",
    "value" : "fv-az1245-423"
  }, {
    "name" : "thread",
    "value" : "2340-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.header_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 5,
        "skipped" : 13,
        "passed" : 42,
        "unknown" : 0,
        "total" : 60
      },
      "items" : [ {
        "uid" : "c95abbdcfb74b5ff",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/149//#testresult/c95abbdcfb74b5ff",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x564f5cb8802a <unknown>\n#1 0x564f5c86e5e0 <unknown>\n#2 0x564f5c8667eb <unknown>\n#3 0x564f5c856a09 <unknown>\n#4 0x564f5c858744 <unknown>\n#5 0x564f5c856cbd <unknown>\n#6 0x564f5c85652f <unknown>\n#7 0x564f5c856422 <unknown>\n#8 0x564f5c854493 <unknown>\n#9 0x564f5c854ada <unknown>\n#10 0x564f5c870f41 <unknown>\n#11 0x564f5c9028c5 <unknown>\n#12 0x564f5c8e2b22 <unknown>\n#13 0x564f5c901d7d <unknown>\n#14 0x564f5c8e28c3 <unknown>\n#15 0x564f5c8b06b3 <unknown>\n#16 0x564f5c8b168e <unknown>\n#17 0x564f5cb52a2b <unknown>\n#18 0x564f5cb569b1 <unknown>\n#19 0x564f5cb3f225 <unknown>\n#20 0x564f5cb57532 <unknown>\n#21 0x564f5cb2438f <unknown>\n#22 0x564f5cb76f28 <unknown>\n#23 0x564f5cb770f3 <unknown>\n#24 0x564f5cb86e7c <unknown>\n#25 0x7f13fb494ac3 <unknown>",
        "time" : {
          "start" : 1728608204051,
          "stop" : 1728608204051,
          "duration" : 0
        }
      }, {
        "uid" : "3e49bd83239552d2",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/322//#testresult/3e49bd83239552d2",
        "status" : "passed",
        "time" : {
          "start" : 1728557099519,
          "stop" : 1728557099593,
          "duration" : 74
        }
      }, {
        "uid" : "b58736698e826eb0",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/321//#testresult/b58736698e826eb0",
        "status" : "passed",
        "time" : {
          "start" : 1728554003417,
          "stop" : 1728554003498,
          "duration" : 81
        }
      }, {
        "uid" : "13d21ab70233923b",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/148//#testresult/13d21ab70233923b",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728522021189,
          "stop" : 1728522021189,
          "duration" : 0
        }
      }, {
        "uid" : "454e3865f438894a",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/320//#testresult/454e3865f438894a",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728470462647,
          "stop" : 1728470462647,
          "duration" : 0
        }
      }, {
        "uid" : "8f2908041f69f9b9",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/319//#testresult/8f2908041f69f9b9",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728467877732,
          "stop" : 1728467877732,
          "duration" : 0
        }
      }, {
        "uid" : "ce83b3c22f689115",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/147//#testresult/ce83b3c22f689115",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728435738628,
          "stop" : 1728435738628,
          "duration" : 0
        }
      }, {
        "uid" : "ed1f8fd4c8e645a4",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/318//#testresult/ed1f8fd4c8e645a4",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728377981002,
          "stop" : 1728377981002,
          "duration" : 0
        }
      }, {
        "uid" : "d257ce5ee974f16b",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/317//#testresult/d257ce5ee974f16b",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728375185979,
          "stop" : 1728375185979,
          "duration" : 0
        }
      }, {
        "uid" : "66b73976d8ddd045",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/316//#testresult/66b73976d8ddd045",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728375071528,
          "stop" : 1728375071528,
          "duration" : 0
        }
      }, {
        "uid" : "b1fb9ec1f078e5aa",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/146//#testresult/b1fb9ec1f078e5aa",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728349151611,
          "stop" : 1728349151611,
          "duration" : 0
        }
      }, {
        "uid" : "63c12091407c9156",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/315//#testresult/63c12091407c9156",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728292730450,
          "stop" : 1728292730450,
          "duration" : 0
        }
      }, {
        "uid" : "6e26d3bca58a0aa",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/314//#testresult/6e26d3bca58a0aa",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728289776510,
          "stop" : 1728289776510,
          "duration" : 0
        }
      }, {
        "uid" : "8c609c129c7d9fbf",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/145//#testresult/8c609c129c7d9fbf",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728263182902,
          "stop" : 1728263182902,
          "duration" : 0
        }
      }, {
        "uid" : "b274f2c270d172fc",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/313//#testresult/b274f2c270d172fc",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728226314357,
          "stop" : 1728226314357,
          "duration" : 0
        }
      }, {
        "uid" : "d0bd22b7fdc55423",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/312//#testresult/d0bd22b7fdc55423",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728221440307,
          "stop" : 1728221440307,
          "duration" : 0
        }
      }, {
        "uid" : "adc731b4c77ec9f5",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/144//#testresult/adc731b4c77ec9f5",
        "status" : "passed",
        "time" : {
          "start" : 1728176837471,
          "stop" : 1728176837548,
          "duration" : 77
        }
      }, {
        "uid" : "5ce61d3a7685061c",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/311//#testresult/5ce61d3a7685061c",
        "status" : "passed",
        "time" : {
          "start" : 1728117473820,
          "stop" : 1728117473898,
          "duration" : 78
        }
      }, {
        "uid" : "e3857e32d54471ab",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/310//#testresult/e3857e32d54471ab",
        "status" : "passed",
        "time" : {
          "start" : 1728108863917,
          "stop" : 1728108863993,
          "duration" : 76
        }
      }, {
        "uid" : "4896f25903368e94",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/143//#testresult/4896f25903368e94",
        "status" : "passed",
        "time" : {
          "start" : 1728090120094,
          "stop" : 1728090120170,
          "duration" : 76
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "16015c77b960bd4e.json",
  "parameterValues" : [ ]
}